packages/vue/src/components/organisms/SfAccordion/SfAccordion.vue

```diff
@@ -1,6 +1,5 @@
 <script src="./SfAccordion.js"></script>
 <template lang="html" src="./SfAccordion.html"></template>
 <style lang="scss">
-@import "../../../utilities/transitions/transitions.scss";
```

Comment:
I think we need transitions here
https://github.com/DivanteLtd/storefront-ui/blob/f24a0a14617a79f0fffecd6e0039c3cd133e6bac/packages/vue/src/components/organisms/SfAccordion/_internal/SfAccordionItem.html#L18